test/parse-regression.sh

```diff
@@ -10,7 +10,7 @@ LC_ALL=C
 export LANG LANGUAGE LC_ALL
 for fn in "${srcdir}"/testdata/*.jpg ; do
     ./test-parse "${fn}" > "${TMPLOG}"
-    if ! diff "${fn}".parsed "${TMPLOG}"; then
+    if ! diff -b "${fn}".parsed "${TMPLOG}"; then
```

Comment:
Whitespace can be significant in some cases, so I don't really want to disable checking.